magefile.go

```diff
@@ -68,6 +68,7 @@ func init() {
 // Build builds the Beat binary.
 func Build() error {
 	args := mage.DefaultBuildArgs()
+	args.InputFiles = []string{"./x-pack/apm-server"}
```

Comment:
This is necessary so that the elastic-agent code is included?

Reply:
Correct, otherwise it just runs `go build`, and builds the program in the repository root.